components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -48,6 +48,17 @@ type NatsConfig struct {
 	// Prefix for the JetStream stream subjects filter.
 	// It will be overridden by non-empty EventTypePrefix.
 	JSStreamSubjectPrefix string `envconfig:"JS_STREAM_SUBJECT_PREFIX" required:"true"`
+
+	// Deliver policy determines for a consumer where in the stream it wants to start receiving messages
+	// (more info https://docs.nats.io/nats-concepts/jetstream/consumers#deliverpolicy-optstartseq-optstarttime):
+	// - all: The consumer will start receiving from the earliest available message.
+	// - last: When first consuming messages, the consumer will start receiving messages with the last message
```

Comment:
```suggestion
	// - last: When first consuming messages, the consumer starts receiving messages with the latest message.
```
I assume "last message" is the latest one?

Reply:
Done.